### PR TITLE
changes

### LINKED ISSUE #2153: [Handbook] Update the Repository Overview, Projects, & Mentorship Programs
**Current Behavior**
These sections are missing some content from the newly updated Handbook draft.

**Desired Behavior**
Update the Repository Overview, Projects & Mentorship Programs section with the details from the [new draft](https://docs.google.com/document/d/1vxc8MLhDetIlBa1DCUTMbYLGWL4Df4K1xB_Ov5oUK7o/edit#heading=h.vtdtai5j3tq7) of the handbook.

---
**Screenshots / Mockups**

**Alternatives**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by moazzam07**
Signed-off-by: Mohammed Moazzam Khan <kmohdmoazzam70@gmail.com>

**Description**

This PR fixes #2153 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 60b94fd119791843b65410e66520b4d830643c85 at: https://614f0cb307a4540ed6b0258b--layer5ng.netlify.app

File: src/sections/Community/Handbook/projects.js [hunk 1]
```diff
@@ -16,19 +19,38 @@ const Maintainer= () => {
...
+                <li><h4><a href="https://meshery.io/">Meshery.io :</a></h4> </li>
+                <p>Website for Meshery, the multi-mesh manager. <br/><a href="https://github.com/meshery/meshery.io"><img className="channels-img" src={github} alt="github"/></a> </p>
+                <li><h4><a href="https://layer5.io/service-mesh-management/meshery/meshery-operator">Meshery-operator :</a></h4> </li>
+                <p>Meshery Operator is the multi-service mesh operator and implementation of MeshSync. <br/><a href="https://github.com/meshery/meshery-operator"><img className="channels-img" src={github} alt="github"/></a> </p>
+              </ul>
+            </p>
+            <h3><a href="https://smp-spec.io/"><img className="channels-img" src={smp} /> Service Mesh Performance</a></h3>
```
Comment (debo19) [inline src/sections/Community/Handbook/projects.js:44]: smpicon
This should be changed to the SMP logo. Check if you're importing from a different address.

File: src/sections/Community/Handbook/projects.js [hunk 2]
```diff
@@ -16,19 +19,38 @@ const Maintainer= () => {
...
-            <h3>3. Service Mesh Performance</h3>
-            <p>The Service Mesh Performance (SMP) is a vendor-neutral specification for capturing details of environment and infrastructure details, service mesh and it’s configuration, service/application details, and bundling of statistical analysis of the result.</p>
-            <h3>4. GetNightHawk</h3>
-            <p>This Is a Layer 7 (HTTP/HTTPS/HTTP2) performance characterization tool.      Nighthawk is Envoy’s load generator and is written in C++. Meshery integrates Nighthawk as one of (currently) three choices of load generator for characterizing and managing the performance of service meshes and their workloads.</p>
+            <p><h3><a href="https://layer5.io/"><img margin="10px" width="4%" align="center" align="left" src={five}/>Layer5</a></h3></p>
```
Comment (debo19) [inline src/sections/Community/Handbook/projects.js:22]: layer5align
Centre the text between the 5 , like you did with the other logos.

**Review by debo19 — CHANGES_REQUESTED** [review]
responsive
The icons are not responsive, they become unaligned and too small in mobile view. Please fix that.

**Comment by debo19** [comment]
@moazzam07 work on the same branch so that you can commit the changes here itself and don't have to create a new PR.

**Pull request closed without merging**